memory/ndctl.py

```diff
@@ -432,6 +432,27 @@ def test_check_numa(self):
             if len(self.plib.run_ndctl_list('-r %s -R -U %s' % (reg, numa))) != 1:
                 self.fail('Region mismatch between ndctl and sys interface')
 
+    @avocado.fail_on(pmem.PMemException)
+    def test_check_ns_numa(self):
+        region = self.get_default_region()
+        self.plib.disable_namespace()
+        self.plib.destroy_namespace()
+        if (not self.plib.is_region_legacy(region)):
+            for _ in range(3):
+                self.plib.create_namespace(
+                    region=region, mode='fsdax', size='128M')
+
+        namespaces = self.plib.run_ndctl_list('-N -r %s' % region)
+        if not os.path.exists('/sys/bus/nd/devices/namespace0.0/numa_node'):
+            self.fail("Numa node entries not found!")
```

Comment:
is this should cancel or fail as if basic requirement not there test should cancel that's what I feel

Reply:
It is a bug if we do not find the entry after enabling namespace.

Reply:
ah sure